### DISCUSSION TITLE
Burst Slow Motion Trigger For FPV Drones

### DISCUSSION THREAD

**Discussion by adityanayak28** [gopro/labs#1068] [Feature Requests]
Hi Everyone, is it possible to add a 'Trigger' Feature which would trigger the Burst Slow Motion once or in continuous mode.

Since the burst slow motion has a limitation of 5s and 15s, i wonder if it's possible to trigger it mid air on my drone using GoPro's Gyro/IMU. 
For example if i do a 360 roll, pitch or yaw on my drone it starts the Burst slow motion or it starts after a countdown of some seconds; Then we get the shot and it ends the burst slow motion or it keeps shooting in continuous mode until we do certain trick to trigger it to stop or maybe you can guys can give some ideas to trigger it in a better way.
Since the rise of Freefly Ember Cameras for ultra slow motion, it would be really helpful to nail some crazy shots using this small camera.

**Comment by dnewman-gpsw** [comment]
Yes there are a few triggers you could use.  

There is a new system variable for each axis of the accelerometer.  ```=Aa:Z```   this will load the Z axis accelerometer value into variable A, and when A is negative, you camera/drone is upside down.  That would be a clear trigger.  You could use a gyro axis to look for flips too.

Try this with holding and inverting you camera as demo:
![image](https://github.com/user-attachments/assets/0de01d9d-5750-4dfe-83cc-c3d482f38b15)

The next more advanced would be do use a GEO fence, GPS location as a trigger. If you know you want ultra slowmo at a particular point, you can preload GPS locations, although the precision maybe insufficient for some applications. Instead you use the above Accelerometer inverted trigger, to mark the current GPS location (more precise.) Then any time you flew through that GPS location, it would trigger the slow motion (or whatever.)  I've never done some this complex, but it should work.  You would invert at the point of interest once, then fly through that location for the slow motion captures.

```$FRMA=0,0```  <- Reset Location marker A, and load the current GPS position into variable c:A.

the condition would be something like ```>c:A50"Slowmo"```  if within 50 meters of location A, do "Slowmo".  

Needs a lot of experimenting.
